plugin/evm/syncervm_test.go

```diff
@@ -331,14 +333,17 @@ func createSyncServerAndClientVMs(t *testing.T, test syncTest, numBlocks int) *s
 	// override serverAtomicTrie's commitInterval so the call to [serverAtomicTrie.Index]
 	// creates a commit at the height [syncableInterval]. This is necessary to support
 	// fetching a state summary.
```

Comment:
let's move this comment above creating the config for `GenesisVMWithUTXOs` on line 287